kpops/cli/main.py

```diff
@@ -28,6 +28,8 @@
 
     from kpops.components.base_components import PipelineComponent
 
+ENV_PREFIX = KpopsConfig.model_config.get("env_prefix")
```

Comment:
This is already [defined in the config](https://github.com/bakdata/kpops/blob/v3/kpops/config.py#L17), you could import the constant from there